src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -702,7 +702,7 @@ private void testDownloadAction() {
                                               .withUserId("CFResultsUiT.instr");
         browser.driver.get(reportUrl.toAbsoluteString());
         String afterReportDownloadUrl = browser.driver.getCurrentUrl();
-        assertFalse(reportUrl.equals(afterReportDownloadUrl));
+        assertFalse(reportUrl.toString().equals(afterReportDownloadUrl));
```

Comment:
@LimeTheCoder, changing to assertEquals caused the CI tests to fail. Should I change it back to use assertFalse?